src/aks-preview/azcli_aks_live_test/scripts/setup_venv.sh

```diff
@@ -28,7 +28,7 @@ setupAZ(){
     ext_repo=${2:-""}
 
     # install azdev, used later to install azcli and extension
-    pip install azdev==0.1.32
+    pip install azdev
```

Comment:
I think it's better to pin to one version (higher version with the fix).

Reply:
Good point, fixed in commit [#8f24632](https://github.com/Azure/azure-cli-extensions/pull/4072/commits/8f24632c4d680dd07fab8a6fc8a0b331decc88c6).